### PR TITLE
Add support for sign specifiers in number formats.

### PR DESCRIPTION
**Pull request by anntzer**
Closes https://github.com/r1chardj0n3s/parse/issues/88.

### CODE REVIEW

File: test_parse.py [hunk 1]
```diff
@@ -205,6 +205,17 @@ def test_typed(self):
...
         self.assertEqual(r.fixed, ('12', 'people'))
 
+    def test_sign(self):
+        # sign is ignored
+        r = parse.parse('Pi = {:.7f}', 'Pi = 3.1415926')
+        self.assertEqual(r.fixed, (3.1415926,))
+        r = parse.parse('Pi = {:+.7f}', 'Pi = 3.1415926')
+        self.assertEqual(r.fixed, (3.1415926,))
+        r = parse.parse('Pi = {:-.7f}', 'Pi = 3.1415926')
+        self.assertEqual(r.fixed, (3.1415926,))
```
Comment (r1chardj0n3s) [inline test_parse.py:215]: I can't help thinking that the parsed value should be negative here.

**Review by r1chardj0n3s — CHANGES_REQUESTED** [review]

File: test_parse.py (hunk 1 above)
Comment (anntzer) [reply test_parse.py:215]: I'm not sure what you mean?  A `-` in the format string just means "use a minus for negative values, and nothing for positive values":
```
In [1]: "{:-.3f}".format(+1.23), "{:-.3f}".format(-1.23)
Out[1]: ('1.230', '-1.230')
```

Comment (r1chardj0n3s) [reply test_parse.py:215]: Oof. That's what I get for trying to do some speed-reviewing of PRs :( Indeed, that looks fine.

**Comment by anntzer** [comment]
Any chance for a new release with this patch?
Thanks in advance.

**Comment by wimglenn** [comment]
@anntzer  1.19.1 is released with this fix.  Apologies for the delay!

**Pull request merged**